test/System.IdentityModel.Tokens.Jwt.Tests/JwtSecurityTokenHandlerTests.cs

```diff
@@ -463,6 +464,99 @@ public void InboundOutboundClaimTypeMapping()
             RunClaimMappingVariation(jwt, handler, validationParameters, expectedClaims: expectedClaims, identityName: null);
         }
 
+#pragma warning disable CS3016 // Arrays as attribute arguments is not CLS-compliant
+        [Theory, MemberData("ReadJwtTokenTheoryData")]
+#pragma warning restore CS3016 // Arrays as attribute arguments is not CLS-compliant
+        public void ReadJwtToken(JwtTheoryData theoryData)
+        {
+            TestUtilities.WriteHeader($"{this}.ReadJwtToken", theoryData);
```

Comment:
Use the new method: TestUtilities.WriteHeader that returns a CompareContext

Reply:
fixed